frontend/src/metabase/nav/containers/MainNavbar/MainNavbar.styled.tsx

```diff
@@ -12,33 +11,21 @@ import { Box, type BoxProps, Icon } from "metabase/ui";
 import { SidebarLink } from "./SidebarItems";
 import { ExpandToggleButton } from "./SidebarItems/SidebarItems.styled";
 
-const openSidebarCSS = css`
-  width: ${NAV_SIDEBAR_WIDTH};
-  border-inline-end: 1px solid var(--mb-color-border);
-
-  ${breakpointMaxSmall} {
-    width: 90vw;
-  }
-`;
-
-const closeSidebarCSS = css`
-  opacity: 0;
-`;
-
 export const Sidebar = styled.aside<{ isOpen: boolean }>`
-  width: 0;
+  ${({ isOpen }) => (isOpen ? "" : "display: none")};
```

Comment:
The sidebar once had a transition, which required toggling its display by changing its width. This kept its buttons in the tab order even when it was closed.